### PR TITLE
Improve d3 charts and tools

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -122,6 +122,20 @@
...
+                * Sketch:
+                    - Add 'creme.d3NumericDataInfo' that create stats from data : min, max, only integers, value range, average.
+                    - Add 'creme.d3NumericFormat' & 'creme.d3NumericAxisFormat' to find the best format from data information.
+                    - Add 'creme.d3Scroll' renderer to handle horizontal or vertical scrolling of an svg node
+                    - creme.D3BarChart: Horizontal scrolling if a minimum bar width is reached.
+                    - creme.D3DonutChart:
+                        - Vertical scrolling of legend on overflow.
+                        - Slice text is visible only on mouse hover if it is too small.
+                    - Fix an issue in 'creme.d3LegendRow' renderer visible in D3TubeChart (legend without labels).
+                    - creme.d3ColorRange now accept a function. Useful for computed ranges.
```
Comment (genglert) [inline CHANGELOG.txt:134]: You put this part in the "deprecations" section. Add a "Apps" section at the end of "non breaking changes" & move this part to it.

File: CHANGELOG.txt [hunk 2]
```diff
@@ -122,6 +122,20 @@
...
+                * Reports:
+                    - creme.D3TubeChart:
+                        - Horizontal scrollog of legend & textwrap.
+                        - Bar text is visible only on mouse hover if it is too small.
```
Comment (genglert) [inline CHANGELOG.txt:138]: A small explanation of the improvements in the "users side" part would be better.
scrollog => scrolling I suppose.

File: creme/reports/static/icecream/reports/css/reports.css [hunk 3]
```diff
@@ -249,3 +249,28 @@ button.reports-chart-selectors-refresh {
...
+    cursor: pointer;
+}
+
+.tube-chart .bar rect:hover {
+    opacity: 0.8;
+}
+
+.tube-chart .bar:not(:hover) .hidden-label {
+    visibility: hidden;
+}
```
Comment (genglert) [inline creme/reports/static/icecream/reports/css/reports.css:276]: Not in "Chantilly" theme?

**Review by genglert — CHANGES_REQUESTED** [review]

File: creme/reports/static/icecream/reports/css/reports.css (hunk 3 above)
Comment (joehybird) [reply creme/reports/static/icecream/reports/css/reports.css:276]: good catch :)

**Comment by genglert** [comment]
Please edit the PR to target the branch 'v2.5-rc6'.

**Pull request merged**